hack/observability/visualizer/values.yaml

```diff
@@ -0,0 +1,12 @@
+replicas: 1
+
+image:
+  repository: ghcr.io/jont828
+  name: cluster-api-visualizer
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.1.0-alpha.2"
```

Comment:
I'm ok with this, but in the future I would like the chart to always give us the latest version, so we are not required to keep this value up to date

Reply:
Sounds good, I'm working towards a v1 of the app with the clusterctl changes from CAPI 1.2 so I'll rework the Helm chart then.